source/components/composing-components.md

```diff
@@ -1,8 +1,11 @@
-Components really shine when you use them to their full potential, that is, by composing them with one another.
-Take for example the `<ul>` element, and the fact that only `<li>` elements are appropriate as children.
-If we want the same type of behavior, then we have to compose our components.
+Components organize applications best when leveraged with composition.
+The phrase "data-down", which is used often when discussing Ember's
+rendering layer, means to treat a user interface as if it were built from
+pure functions. By pure function, we mean a function that given the same input
+will always result in the same behavior.
```

Comment:
I was unsold on the comparison to `<ul><li>`. HTML is markup, and since `ul` passes nothing to `li` I don't see where composition comes into play. That `ul` has

> only `<li>` elements are appropriate as children

is not composition, but just seems like contextual behavior. If to goal is to compare component composition to something the user already understands, comparing it to object creation or just function calls seems more ideal to me.